guides/06_client-libraries/02_getting-started-with-the-js-client.md

```diff
@@ -0,0 +1,251 @@
+# Getting Started with the Gradio JavaScript client 
+
+Tags: CLIENT, API, SPACES
+
+
+The Gradio JavaScript client makes it very easy to use any Gradio app as an API. As an example, consider this [Hugging Face Space that transcribes audio files](https://huggingface.co/spaces/abidlabs/whisper) that are recorded from the microphone.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/gradio-guides/whisper-screenshot.jpg)
+
+Using the `@gradio/client` library, we can easily use the Gradio as an API to transcribe audio files programmatically.
+
+Here's the entire code to do it:
+
+```js
+import { client } from "@gradio/client";
+
+const response = await fetch(
+  "https://audio-samples.github.io/samples/mp3/blizzard_unconditional/sample-0.mp3"
+);
+const audio_file = await response.blob();
+
+const app = client("abidlabs/whisper");
+const transcription = app.predict("/predict", [audio_file]);
+
+// "This is a test of the whisper speech recognition model."
+```
+
+The Gradio client works with any hosted Gradio app, whether it be an image generator, a text summarizer, a stateful chatbot, a tax calculator, or anything else! The Gradio Client is mostly used with apps hosted on [Hugging Face Spaces](https://hf.space), but your app can be hosted anywhere, such as your own server.
+
+**Prequisites**: To use the Gradio client, you do *not* need to know the `gradio` library in great detail. However, it is helpful to have general familiarity with Gradio's concepts of input and output components.
+
+## Installation
+
+If you already have a recent version of `gradio`, then the `gradio_client` is included as a dependency. 
+
+Otherwise, the lightweight `@gradio/client` package can be installed from the npm registry with a package manager of your choice and support node version 18 and above:
+
+```bash
+$ pnpm add @gradio/client
```

Comment:
The README says we should do "npm i -D @gradio/client". Probably both work, but we should pick 1 in our docs?